README.md

```diff
@@ -147,7 +147,9 @@ open http://localhost:8080
 ## Browser Support
 
 - Chrome
-- Safari 8+
+- Safari
 - Firefox
 - IE 11/Edge
 - Opera
+
+In general our policy is to support the last two versions of every major browser.
```

Comment:
I'd move this to the top and use more concrete language, e.g. 

> We officially support the last two versions of every major browser. Specifically, we test on the following browsers:

I'd also maybe add to more items to that list for Mobile Safari and Chrome on Android